bots/images/scripts/fedora.setup

```diff
@@ -27,16 +29,21 @@ if [ $(uname -m) = x86_64 ]; then
     HAVE_KUBERNETES=1
 fi
 
+HAVE_DOCKER=
+if type docker >/dev/null 2>&1; then
+    HAVE_DOCKER=1
+fi
+
 # We install all dependencies of the cockpit packages since we want
 # them to not spontaneously change from one test run to the next when
 # the distribution repository is updated.
 #
 COCKPIT_DEPS="\
-atomic \
 device-mapper-multipath \
-docker \
 etcd \
+glibc-all-langpacks \
```

Comment:
Why do we need that?

Reply:
Otherwise the localization tests fail. These are not installed by default on f31.

Reply:
just out of interest, I thought we only tested localization on F30 (but surely we can switch to F31).